R/plot_MutationTimeR.R

```diff
@@ -3,40 +3,39 @@
 #' @description This function plots the timing information from MutationTimeR for SSMs and CNAs.
 #'
 #' @param this_sample_metadata Metadata with one row containing your sample details
-#' @param projection Genome build projection (e.g., "hg38" or "grch37").
+#' @param timed_cna Timed CNAs from `GAMBLR.results::get_timed_mutations`
+#' @param timed_ssm Timed SSMs from `GAMBLR.results::get_timed_mutations`
 #' @param verbose Set to TRUE for a chattier experience. Default is FALSE.
 #' @import ggplot2 dplyr ggpubr
```

Comment:
I realize this function doesn't have a @return line or I'm missing it because it's late on a Friday afternoon. Anyway, please make sure it has one that describes what the function returns.